defaults/main.yml

```diff
@@ -9,6 +9,7 @@ nodejs_major_version: 12
 nodejs_version: 12.16.2
 rails_app_use_webpack: true
 yarn_version: v1.13.0
+redis_version: 5.0.10
```

Comment:
I'm would not add the redis default version here. It would usually belongs in the role that installs redis, but since that is an external role, I'd just keep it in the playbook.

Reply:
@bibliotechy Okay, I took it out.